test/src/main/java/io/strimzi/test/BaseITST.java

```diff
@@ -0,0 +1,210 @@
+/*
+ * Copyright 2018, Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.test;
+
+import io.fabric8.kubernetes.client.DefaultKubernetesClient;
+import io.strimzi.test.k8s.KubeClient;
+import io.strimzi.test.k8s.KubeClusterResource;
+import io.strimzi.test.timemeasuring.Operation;
+import io.strimzi.test.timemeasuring.TimeMeasuringSystem;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.TestInfo;
+import org.junit.jupiter.api.TestInstance;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Stack;
+import java.util.stream.Collectors;
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+public class BaseITST {
+
+    private static final String CO_INSTALL_DIR = "../install/cluster-operator";
+
+    private static final Logger LOGGER = LogManager.getLogger(BaseITST.class);
+    protected static final String CLUSTER_NAME = "my-cluster";
+
+    public static final KubeClusterResource CLUSTER = new KubeClusterResource();
+    protected static final DefaultKubernetesClient CLIENT = new DefaultKubernetesClient();
+    public static final KubeClient<?> KUBE_CLIENT = CLUSTER.client();
+    private static final String DEFAULT_NAMESPACE = KUBE_CLIENT.defaultNamespace();
+
+    public String clusterOperatorNamespace = DEFAULT_NAMESPACE;
```

Comment:
I guess access for `clusterOperatorNamespace` and `bindingsNamespaces` should be `protected` and access for `deploymentNamespaces` should be `privat`